apmpackage/apm/data_stream/internal_metrics_transaction/fields/faas_metrics.yml

```diff
@@ -0,0 +1,20 @@
+- name: faas.billed_duration
+  type: long
+  metric_type: counter
+  unit: ms
+  description: The number of milliseconds for which the FaaS end user is billed.
+- name: faas.coldstart_duration
+  type: float
+  metric_type: counter
+  unit: ms
+  description: The number of milliseconds elapsed during cold start.
+- name: faas.duration
+  type: float
+  metric_type: counter
+  unit: ms
+  description: The total number of milliseconds elapsed during the execution of the FaaS invocation.
+- name: faas.timeout
+  type: long
+  metric_type: counter
+  unit: ms
+  description: The FaaS invocation timeout, expressed in milliseconds.
```

Comment:
Are these relevant to transaction metrics? Doesn't look like it, looking at txmetrics code.

Reply:
absolutely right, I'll delete these.